src/Shared/UnitTests/MockLogger.cs

```diff
@@ -416,7 +416,7 @@ static private ResourceManager EngineResourceManager
             {
                 if (s_engineResourceManager == null)
                 {
-                    s_engineResourceManager = new ResourceManager("Microsoft.Build.Resources.Strings", typeof(ProjectCollection).GetTypeInfo().Assembly);
+                     s_engineResourceManager = new ResourceManager("Microsoft.Build.Strings", typeof(ProjectCollection).GetTypeInfo().Assembly);
```

Comment:
Pretty sure this is the right change, right? Combining `GetTypeInfo` from xplat with the rename from master?